### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by raeich**
This PR is a test!

### CODE REVIEW

**Comment by raeich** [comment]
Hey @mfaade! Let me know what you think about these top 5.

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# Test
\ No newline at end of file
+# Test
+
+Hey world, here is a list of my top 5 foods.
```
Comment (mjfaade10) [inline README.md:3]: Hey! Can you change this to top 5 movies?

**Review by mjfaade10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (raeich) [reply README.md]: @mjfaade10 this has been updated!

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
-# Test
\ No newline at end of file
+# Test
+
+Hey world, here is a list of my top 5 movies.
+
+1. Garden State
+2. Superbad
```
Comment (mjfaade10) [inline README.md:6]: Love this movie!

File: README.md [hunk 3]
```diff
@@ -1 +1,9 @@
...
+3. 25th Hour
```
Comment (mjfaade10) [inline README.md:7]: I haven't seen this one yet. Will have to check it out 💯

**Review by mjfaade10 — APPROVED** [review]

**Pull request merged**